### ISSUE TITLE
everything, but including dictionary, should have a string representation

### ISSUE THREAD

**Issue by rickspencer3** [influxdata/flux#4454]
When debugging Flux, it can be difficult or impossible to see values that are passed from other API calls. For example, the headers of a response object in the http library are a dictionary.

It would be useful if you could pass in any object to array.from for debugging. For example:

```flux
array.from(rows: [{"headers":string(v: response.headers}])
|> yield(name: "print headers")
```

**Comment by nathanielc** [comment]
I agree however I might suggest something slightly different. In python they have both `str` and `repr` functions. Currently the Flux `string` function behaves more like Python's `str` function. I think we should add a new function called `repr` (or another name) that prints out the structure of the data directly.

How are they different? Take the `bytes`  type as an example. The behavior of `string` for bytes is to treat the data as a sequence of utf-8 char and convert it to a utf-8 encoded string. But not all bytes are utf-8, so it would fail for those cases. The `repr` function could simply print out the byte values as a big long list and not care about utf-8 encoding. 

Thoughts?
